README.md

```diff
@@ -29,30 +29,32 @@ return {
     config = function()
         -- You don't need to set these options.
         require("colorful-menu").setup({
-            ft = {
-                lua = {
+            ls = {
+                lua_ls = {
                     -- Maybe you want to dim arguments a bit.
                     auguments_hl = "@comment",
```

Comment:
While you are at it, maybe fix this typo for the config option while the plugin is still very young?😅

Reply:
Fixed in #16 :)